src/vs/workbench/contrib/extensions/browser/extensionEditor.ts

```diff
@@ -174,6 +174,12 @@ export class ExtensionEditor extends BaseEditor {
 	private extensionChangelog: Cache<string> | null;
 	private extensionManifest: Cache<IExtensionManifest | null> | null;
 
+	// Some action bar items use a webview whose vertical scroll position we track in this array (Readme=0, Changelog=1)
+	private initialScrollProgress: number[] = [];
```

Comment:
Use a map here since it providers better type checking

Reply:
Good suggestion. I have pushed an update containing this.